api.go

```diff
@@ -115,6 +116,7 @@ func (w *Worker) saveImage(image albumImage, folder string) error {
 		return errors.New("unable to find valid image filename, skipping")
 	}
 	dest := fmt.Sprintf("%s/%s", folder, image.Name())
+	dest = strings.Replace(dest, "?", "_", -1)
```

Comment:
this is not related to the current PR. It is, instead, related to https://github.com/tommyblue/smugmug-backup/issues/62
It's better to split the work, so that each one can have its own discussion